src/python/ovms_py_tensor.cpp

```diff
@@ -55,17 +56,32 @@ OvmsPyTensor::OvmsPyTensor(const std::string& name, void* ptr, const std::vector
     }
 }
 
-OvmsPyTensor::OvmsPyTensor(const std::string& name, py::buffer_info bufferInfo) :
+OvmsPyTensor::OvmsPyTensor(const std::string& name, const py::buffer& buffer) :
     name(name),
-    ptr(bufferInfo.ptr),
-    bufferShape(bufferInfo.shape),
-    ndim(bufferInfo.ndim),
-    format(bufferInfo.format),
-    itemsize(bufferInfo.itemsize),
-    strides(bufferInfo.strides) {
+    ptr(),
+    bufferShape(),
+    ndim(),
+    format(),
+    itemsize(),
```

Comment:
Why do you need those default constructors?

Reply:
For the compiler not to brag about the initialization order

Reply:
You can correct the order in class definition